curriculum/challenges/english/25-front-end-development/quiz-basic-css/66ed8fa2f45ce3ece4053eab.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
 
 ---
 
-Placeholder distractor 3
+`flex-width: 50px; flex-height: 50px;`
 
 #### --answer--
 
-Placeholder answer
+`width: 50px; height: 50px;`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which rule is correct to target `<h1>` inside of a `<div>`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`div + h1 {}`
 
 ---
 
-Placeholder distractor 2
+`div ~ h1 {}`
 
 ---
 
-Placeholder distractor 3
+`div > h1 {}`
 
 #### --answer--
 
-Placeholder answer
+`div h1 {}`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which rule is correct to target direct children of a `<footer>`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`footer ~ ul {}`
 
 ---
 
-Placeholder distractor 2
+`footer + ul {}`
 
 ---
 
-Placeholder distractor 3
+`footer ul {}`
 
 #### --answer--
 
-Placeholder answer
+`footer > ul {}`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which rule is correct to target the next sibling of an `<img>`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`img h1 {}`
 
 ---
 
-Placeholder distractor 2
+`img > h1 {}`
 
 ---
 
-Placeholder distractor 3
+`img ~ h1 {}`
 
 #### --answer--
 
-Placeholder answer
+`img + h1 {}`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which rule is correct to target all siblings of an `<img>`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`img > caption {}`
 
 ---
 
-Placeholder distractor 2
+`img caption {}`
 
 ---
 
-Placeholder distractor 3
+`img + caption {}`
 
 #### --answer--
 
-Placeholder answer
+`img ~ caption {}`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What statement is TRUE about `block` elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+Blockline elements don't start on new lines.
 
 ---
 
-Placeholder distractor 2
+Blockline elements only takes up as much as it needs.
 
 ---
 
-Placeholder distractor 3
+Blockline elements come with no margin.
 
 #### --answer--
 
-Placeholder answer
+Blockline elements start on a new line with margin before and after the element.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What statement is TRUE when using the `inline-block` rule?
 
 #### --distractors--
 
-Placeholder distractor 1
+Doesn't allow you to set the height and width.
 
 ---
 
-Placeholder distractor 2
+Top and bottom margins not respected.
 
 ---
 
-Placeholder distractor 3
+Adds a line break after an element.
 
 #### --answer--
 
-Placeholder answer
+Allows you to place elements next to each other while letting you set the height and width.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Given the following selectors, which has the highest priority?
 
 #### --distractors--
 
-Placeholder distractor 1
+`div`
 
 ---
 
-Placeholder distractor 2
+`h1`
 
 ---
 
-Placeholder distractor 3
+`p`
 
 #### --answer--
 
-Placeholder answer
+`#id`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Given the following selectors, which has the lowest priority?
```

Comment:
```suggestion
Given the following selectors, which has the lowest specificity?
```

The answers also don't match. Type selectors have the same specificity - `div` is no more the correct answer than `h1` is.